Index: src/main/webapp/WEB-INF/views/borrow.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/WEB-INF/views/borrow.jsp	(date 1530360485977)
+++ src/main/webapp/WEB-INF/views/borrow.jsp	(date 1530360485977)
@@ -0,0 +1,155 @@
+<%@ taglib prefix="a" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <style>
+        <%@include file='main-style.css' %>
+    </style>
+    <title>Edit</title>
+</head>
+<body>
+<nav class="navbar navbar-expand-lg navbar-light bg-light" >
+    <a class="navbar-brand " href="/">RentCar</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+            <sec:authorize access="isAuthenticated()">
+                <li class="nav-item">
+                    <a class="nav-link" href="/lend">Lend</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/borrow">Borrow</a>
+                </li>
+            </sec:authorize>
+            <sec:authorize access="hasRole('ROLE_ADMIN')">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="/admin" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Admin
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                        <a class="dropdown-item" href="/admin/brand">Brand</a>
+                        <a class="dropdown-item" href="/admin/city">City</a>
+                    </div>
+                </li>
+            </sec:authorize>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+            <sec:authorize access="isAnonymous()">
+                <li><a href="/registration"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+                <li><a href="/login"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+            </sec:authorize>
+            <sec:authorize access="isAuthenticated()">
+                <li><a href="/profile"><span class="glyphicon glyphicon-user"></span> Profile</a></li>
+                <form:form action="/logout">
+                    <a><button class="my-button-like-link"><span class="glyphicon glyphicon-log-in"></span> Logout</button></a>
+                </form:form>
+            </sec:authorize>
+        </ul>
+    </div>
+</nav>
+<div class="header">
+    <div class="container-for-article col-md-5 col-xs-12">
+        <h2 style="color: #2981c0; text-align: center; margin-top: 12%;"><b>There are list of all free cars</b></h2>
+        <h4 style="color: #2981c0; text-align: center;">With filters help you can find car with particular details</h4>
+    </div>
+    <div class="container-for-picture col-md-7 col-xs-12">
+
+    </div>
+</div>
+<div class="main-container-borrow col-md-7 col-xs-12">
+    <div class="container-for-main-filter">
+        <div class="form-row">
+            <h2 style="color: #2981c0"><b>Main filter</b></h2>
+        </div>
+    </div>
+
+    <div class="container-for-free-car">
+        <div class="form-row">
+            <h2 style="color: #2981c0"><b>Free cars</b></h2>
+        </div>
+        <div style="height:390px; overflow:auto;">
+            <a:forEach items="${freeCars}" var="order">
+                <div class="my-particular-order">
+                    City: <b>${order.region}</b> |
+                    Price: <b>${order.pricePerDay}</b> |
+                    Chauffeur: <b>${order.chauffeur}</b> |
+                    Rent from: <b>${order.periodOfRentFrom}</b> |
+                    Rent to: <b>${order.periodOfRentTo}</b>
+                        <a href="/borrow/choose/${order.id}" class="button">Choose</a>
+                </div>
+            </a:forEach>
+        </div>
+    </div>
+</div>
+
+<div class="left-side-container-borrow col-md-5 col-xs-12" >
+    <div class="container-for-filter ">
+        <div class="form-row">
+            <h2 style="color: #2981c0"><b>Additional filters</b></h2>
+        </div>
+    </div>
+    <div class="container-for-wishes " >
+        <div class="form-row">
+            <h2 style="color: #2981c0"><b>Wishes</b></h2>
+        </div>
+        <div style="height:390px; overflow:auto;">
+            <a:forEach items="${selectedOrders}" var="order">
+                <div class="my-particular-order">
+                    <a style="float: right" href="/borrow/delete/${order.driverId}/${order.id}" class="btn btn-warning btn-sm">Delete</a>
+                    <a style="float: right" href="/driver/${order.driverId}" class="btn btn-warning btn-sm">Driver</a>
+                    City: <b>${order.region}</b> |
+                    Price: <b>${order.pricePerDay}</b> |
+                    Chauffeur: <b>${order.chauffeur}</b> <br>
+                    Rent from: <b>${order.periodOfRentFrom}</b> |
+                    Rent to: <b>${order.periodOfRentTo}</b>
+                </div>
+            </a:forEach>
+            <a:forEach items="${reservedOrder}" var="order">
+                <div class="my-finished-trips-for-borrow">
+                    <a style="float: right" href="/driver/${order.driverId}" class="btn btn-warning btn-sm">Driver</a>
+                    <a style="float: right" href="/borrow/complete/${order.id}" class="btn btn-warning btn-sm">Complete</a>
+                    City: <b>${order.region}</b> |
+                    Price: <b>${order.pricePerDay}</b> |
+                    Chauffeur: <b>${order.chauffeur}</b> <br>
+                    Rent from: <b>${order.periodOfRentFrom}</b> |
+                    Rent to: <b>${order.periodOfRentTo}</b>
+                </div>
+            </a:forEach>
+        </div>
+    </div>
+    <div class="container-for-finished-trips" >
+        <div class="form-row">
+            <h2 style="color: #2981c0"><b>Finished trips</b></h2>
+            <a:forEach items="${finishedOrders}" var="order">
+                <div class="my-particular-order">
+                    <a style="float: right" href="/driver/${order.driverId}" class="btn btn-warning btn-sm">Driver</a>
+                    City: <b>${order.region}</b> |
+                    Price: <b>${order.pricePerDay}</b> |
+                    Chauffeur: <b>${order.chauffeur}</b> <br>
+                    Rent from: <b>${order.periodOfRentFrom}</b> |
+                    Rent to: <b>${order.periodOfRentTo}</b>
+                </div>
+            </a:forEach>
+        </div>
+    </div>
+</div>
+
+
+</body>
+</html>
\ No newline at end of file
Index: src/main/webapp/WEB-INF/views/login.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/WEB-INF/views/login.jsp	(date 1526564373617)
+++ src/main/webapp/WEB-INF/views/login.jsp	(date 1526564373617)
@@ -0,0 +1,94 @@
+<%@ taglib prefix="a" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <style>
+        <%@include file='main-style.css' %>
+    </style>
+    <title>Login</title>
+</head>
+<body style="background-image: url(https://www.247carkeysolutions.co.uk/wp-content/uploads/2016/11/carkeys_shutterstock_74617219.jpg);">
+<nav class="navbar navbar-expand-lg navbar-light bg-light" >
+    <a class="navbar-brand " href="/">RentCar</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+            <sec:authorize access="isAuthenticated()">
+                <li class="nav-item">
+                    <a class="nav-link" href="/lend">Lend</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/borrow">Borrow</a>
+                </li>
+            </sec:authorize>
+            <sec:authorize access="hasRole('ROLE_ADMIN')">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="/admin" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Admin
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                        <a class="dropdown-item" href="/admin/brand">Brand</a>
+                        <a class="dropdown-item" href="/admin/city">City</a>
+                    </div>
+                </li>
+            </sec:authorize>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+            <sec:authorize access="isAnonymous()">
+                <li><a href="/registration"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+                <li><a href="/login"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+            </sec:authorize>
+            <sec:authorize access="isAuthenticated()">
+                <li><a href="/profile"><span class="glyphicon glyphicon-user"></span> Profile</a></li>
+                <form:form action="/logout">
+                    <a><button class="my-button-like-link"><span class="glyphicon glyphicon-log-in"></span> Logout</button></a>
+                </form:form>
+            </sec:authorize>
+        </ul>
+    </div>
+</nav>
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-1 col-sm-5 col-lg-6"></div>
+            <div class="my-login-form col-xs-10 col-sm-6 col-lg-4">
+                <form:form  action="/login" method="post" >
+                    <div class="form-group row">
+                        <c:if test="${param.fail}">
+                            <div class="fail-to-authorize">
+                                Fail to authorize
+                            </div>
+                        </c:if>
+                    </div>
+                    <div class="form-group row">
+                        <input name="login" class="form-control"  placeholder="Input email"/>
+                    </div>
+                    <div class="form-group row">
+                        <input type="password" name="password" class="form-control"  placeholder="Input password"/>
+                    </div>
+                    <div class="form-group row">
+                        <label><span style="color: aliceblue">Remember me</span>  <input type="checkbox" name="rememberMe"></label>
+                    </div>
+                    <div class="form-group row">
+                        <button type="submit" class="btn btn-success mb-2">Login</button>
+                        <a href="/registration" class="btn btn-info mb-2">Sign Up</a>
+                    </div>
+                </form:form>
+            </div>
+        </div>
+    </div>
+</body>
+</html>
\ No newline at end of file
Index: src/main/java/ua/entity/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/entity/User.java	(date 1529936672803)
+++ src/main/java/ua/entity/User.java	(date 1529936672803)
@@ -0,0 +1,55 @@
+package ua.entity;
+
+import ua.entity.enums.Role;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="_user")
+public class User extends AbstractEntity {
+
+
+    private String email;
+
+    private String password;
+
+    private Role role;
+
+    @OneToOne(mappedBy="user", cascade = CascadeType.PERSIST)
+    private Driver driver;
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public Role getRole() {
+        return role;
+    }
+
+    public void setRole(Role role) {
+        this.role = role;
+    }
+
+    public Driver getDriver() {
+        return driver;
+    }
+
+    public void setDriver(Driver driver) {
+        this.driver = driver;
+    }
+}
\ No newline at end of file
Index: src/main/java/ua/service/FileWriter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/service/FileWriter.java	(date 1530432501439)
+++ src/main/java/ua/service/FileWriter.java	(date 1530432501439)
@@ -0,0 +1,7 @@
+package ua.service;
+
+import org.springframework.web.multipart.MultipartFile;
+
+public interface FileWriter {
+    String write(MultipartFile file, String email);
+}
Index: src/main/java/ua/model/request/FileRequest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/model/request/FileRequest.java	(date 1530539256415)
+++ src/main/java/ua/model/request/FileRequest.java	(date 1530539256415)
@@ -0,0 +1,16 @@
+package ua.model.request;
+
+import org.springframework.web.multipart.MultipartFile;
+
+public class FileRequest {
+
+    private MultipartFile file;
+
+    public MultipartFile getFile() {
+        return file;
+    }
+
+    public void setFile(MultipartFile file) {
+        this.file = file;
+    }
+}
Index: src/main/java/ua/controller/PageOfDriverController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/controller/PageOfDriverController.java	(date 1530197100639)
+++ src/main/java/ua/controller/PageOfDriverController.java	(date 1530197100639)
@@ -0,0 +1,28 @@
+package ua.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import ua.service.DriverService;
+import ua.service.RentService;
+
+import java.security.Principal;
+
+@Controller
+@RequestMapping("/driver")
+public class PageOfDriverController {
+
+    private final DriverService service;
+
+    public PageOfDriverController(DriverService service) {
+        this.service = service;
+    }
+
+    @GetMapping("/{id}")
+    public String show(@PathVariable Integer id, Model model) {
+        model.addAttribute("driverInfo", service.pageOfDriver(id));
+        return "info-about-driver";
+    }
+}
Index: src/main/java/ua/service/impl/FileWriterImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/service/impl/FileWriterImpl.java	(date 1530544880701)
+++ src/main/java/ua/service/impl/FileWriterImpl.java	(date 1530544880701)
@@ -0,0 +1,53 @@
+package ua.service.impl;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import ua.entity.Driver;
+import ua.repository.DriverRepository;
+import ua.service.FileWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.UUID;
+
+
+@Service
+public class FileWriterImpl implements FileWriter {
+
+    private final DriverRepository repository;
+
+    public FileWriterImpl(DriverRepository repository) {
+        this.repository = repository;
+    }
+
+    @Value("${file.path}")
+    private String path;
+
+    @Override
+    public String write(MultipartFile file, String email) {
+        String filename = file.getOriginalFilename();
+        System.out.println(filename+"-------------");
+        String [] split = filename.split("\\*.\\*");
+        String expansion = split[split.length-1];
+        System.out.println(expansion+"-expansion----------- ");
+        String newName = UUID.randomUUID().toString().concat(".").concat(expansion);
+        File oldFile =new File(path, file.getOriginalFilename());
+        File newFile =new File(path, newName);
+        if(oldFile.renameTo(newFile)){
+            System.out.println("Rename succesful");
+        }else{
+            System.out.println("Rename failed");
+        }
+
+        try {
+            file.transferTo(newFile);
+            Driver driver = repository.findAuthorizeDriverByEmail(email);
+            driver.setPhotoURL(newName);
+            repository.save(driver);
+        } catch (IOException e) {
+            return null;
+        }
+        return newFile.toString();
+    }
+}
Index: src/main/webapp/WEB-INF/views/info-about-driver.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/WEB-INF/views/info-about-driver.jsp	(date 1530356402904)
+++ src/main/webapp/WEB-INF/views/info-about-driver.jsp	(date 1530356402904)
@@ -0,0 +1,103 @@
+<%@ taglib prefix="a" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <style>
+        <%@include file='admin-style.css' %>
+    </style>
+</head>
+<body>
+<nav class="navbar navbar-expand-lg navbar-light bg-light" >
+    <a class="navbar-brand " href="/">RentCar</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+            <sec:authorize access="isAuthenticated()">
+                <li class="nav-item">
+                    <a class="nav-link" href="/lend">Lend</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/borrow">Borrow</a>
+                </li>
+            </sec:authorize>
+            <sec:authorize access="hasRole('ROLE_ADMIN')">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="/admin" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Admin
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                        <a class="dropdown-item" href="/admin/brand">Brand</a>
+                        <a class="dropdown-item" href="/admin/city">City</a>
+                    </div>
+                </li>
+            </sec:authorize>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+            <sec:authorize access="isAnonymous()">
+                <li><a href="/registration"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+                <li><a href="/login"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+            </sec:authorize>
+            <sec:authorize access="isAuthenticated()">
+                <li><a href="/profile"><span class="glyphicon glyphicon-user"></span> Profile</a></li>
+                <form:form action="/logout">
+                    <a><button class="my-button-like-link"><span class="glyphicon glyphicon-log-in"></span> Logout</button></a>
+                </form:form>
+            </sec:authorize>
+        </ul>
+    </div>
+</nav>
+    <div class="container">
+        <div class="row">
+            <div class="col-3"></div>
+            <div class="col-4">
+                <table class="table table-bordered table-dark">
+                    <tbody>
+                        <tr>
+                            <td class="text-right">Name</td>
+                            <td>${driverInfo.name}</td>
+                        </tr>
+                        <tr>
+                            <td class="text-right">Surname</td>
+                            <td>${driverInfo.surname}</td>
+                        </tr>
+                        <tr>
+                            <td class="text-right">Phone</td>
+                            <td>${driverInfo.phone}</td>
+                        </tr>
+                        <tr>
+                            <td class="text-right">Age</td>
+                            <td>${driverInfo.age}</td>
+                        </tr>
+                        <tr>
+                            <td class="text-right">Experience</td>
+                            <td>${driverInfo.experience}</td>
+                        </tr>
+                        <tr>
+                            <td class="text-right">Count of trips</td>
+                            <td>${driverInfo.countOfTrips}</td>
+                        </tr>
+                        <tr>
+                            <td class="text-right">Place of birth</td>
+                            <td>${driverInfo.placeOfBirth}</td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </div>
+</body>
+</html>
\ No newline at end of file
Index: src/main/java/ua/repository/RentRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/repository/RentRepository.java	(date 1530360836645)
+++ src/main/java/ua/repository/RentRepository.java	(date 1530360836645)
@@ -0,0 +1,60 @@
+package ua.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import ua.entity.Driver;
+import ua.entity.InfoAboutRent;
+import ua.entity.enums.Status;
+import ua.model.view.CarView;
+import ua.model.view.DriverView;
+import ua.model.view.OrderView;
+
+import java.util.List;
+
+public interface RentRepository extends JpaRepository<InfoAboutRent,Integer> {
+
+//    -------------------------query for LendController --------------------------------
+    @Query("select d from Driver d join d.user u where  u.email = ?1")
+    Driver findDriverByUserEmail(String email);
+
+    //User can add only one active order, for next order user should finish previous
+    @Query("select inf from InfoAboutRent inf join inf.car c join c.driver d join d.user u where u.email=?1 and inf.status=?2")
+    InfoAboutRent checkIfOrderIsActive(String email, Status active);
+
+    @Query("select c.name from City c")
+    List<String> findAllCities();
+
+    @Query("select new ua.model.view.OrderView(o.id, ds.id, o.pricePerDay, o.periodOfRentFrom, o.periodOfRentTo, r.name, o.chauffeur, o.status)" +
+            "from InfoAboutRent o join o.drivers ds join o.region r join o.car c join c.driver d join d.user u where u.email=?1 and o.status = ?2 order by o desc")
+    List<OrderView> findFinishedOrders(String email, Status passive);
+
+    @Query("select new ua.model.view.OrderView(o.id, o.pricePerDay, o.periodOfRentFrom, o.periodOfRentTo, r.name, o.chauffeur, o.status)" +
+            "from InfoAboutRent o join o.region r join o.car c join c.driver d join d.user u where u.email=?1 and (o.status =?2 or o.status =?3 or o.status =?4)")
+    OrderView findParticularOrder(String email, Status active, Status reserved, Status completed);
+
+    @Query("select new ua.model.view.DriverView(d.id, d.name, d.surname, d.phone, d.countOfTrips, p.name, d.dateOfBirth, d.experienceBegan)" +
+            " from Driver d join d.placeOfBirth p join d.listOfInfoAboutCar inf where inf.id =?1 and inf.status =?2")
+    List<DriverView> findAllClients(Integer id, Status active);
+
+    @Query("select new ua.model.view.DriverView(d.id, d.name, d.surname, d.phone, d.countOfTrips, p.name, d.dateOfBirth, d.experienceBegan)" +
+            " from Driver d join d.placeOfBirth p join d.listOfInfoAboutCar inf where inf.id =?1 and (inf.status=?2 or inf.status = ?3)")
+    List<DriverView> findReservedOrder(Integer id, Status reserved, Status completed);
+
+    @Query("select d from Driver d where d.id=?1")
+    Driver findOneDriver(Integer id);
+
+    @Query("select o from InfoAboutRent o where o.id=?1")
+    InfoAboutRent findParticularOrderById(Integer id);
+
+//    -----------------------query for BorrowController---------------------
+    @Query("select i from InfoAboutRent i where i.id=?1")
+    InfoAboutRent findOrderById(Integer id);
+
+    @Query("select new ua.model.view.OrderView(o.id,  o.pricePerDay, o.periodOfRentFrom, o.periodOfRentTo, r.name, o.chauffeur, o.status)" +
+            "from InfoAboutRent o join o.region r join o.car c join c.driver d join d.user u where u.email <> ?1 and o.status= ?2 order by o desc")
+    List<OrderView> findAllFreeCars(String email, Status status);
+
+    @Query("select new ua.model.view.OrderView(o.id, ds.id, o.pricePerDay, o.periodOfRentFrom, o.periodOfRentTo, r.name, o.chauffeur, o.status)" +
+            "from InfoAboutRent o join o.drivers ds join o.region r where ds.id = (select d.id from Driver d join d.user u where u.email=?1) and (o.status = ?2 or o.status=?3)  order by o desc")
+    List<OrderView> findOrdersByDriverId(String email, Status active, Status completed);
+}
Index: src/main/webapp/WEB-INF/views/registration.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/WEB-INF/views/registration.jsp	(date 1526564373799)
+++ src/main/webapp/WEB-INF/views/registration.jsp	(date 1526564373799)
@@ -0,0 +1,161 @@
+<%@ taglib prefix="a" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <style>
+        <%@include file='main-style.css' %>
+    </style>
+    <title>Sign Up</title>
+</head>
+<body style="background-image: url(https://www.247carkeysolutions.co.uk/wp-content/uploads/2016/11/carkeys_shutterstock_74617219.jpg);">
+<nav class="navbar navbar-expand-lg navbar-light bg-light" >
+    <a class="navbar-brand " href="/">RentCar</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+            <sec:authorize access="isAuthenticated()">
+                <li class="nav-item">
+                    <a class="nav-link" href="/lend">Lend</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/borrow">Borrow</a>
+                </li>
+            </sec:authorize>
+            <sec:authorize access="hasRole('ROLE_ADMIN')">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="/admin" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Admin
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                        <a class="dropdown-item" href="/admin/brand">Brand</a>
+                        <a class="dropdown-item" href="/admin/city">City</a>
+                    </div>
+                </li>
+            </sec:authorize>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+            <sec:authorize access="isAnonymous()">
+                <li><a href="/registration"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+                <li><a href="/login"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+            </sec:authorize>
+            <sec:authorize access="isAuthenticated()">
+                <li><a href="/profile"><span class="glyphicon glyphicon-user"></span> Profile</a></li>
+                <form:form action="/logout">
+                    <a><button class="my-button-like-link"><span class="glyphicon glyphicon-log-in"></span> Logout</button></a>
+                </form:form>
+            </sec:authorize>
+        </ul>
+    </div>
+</nav>
+    <div class="container">
+        <div class="row">
+            <div class=" col-sm-3"></div>
+            <div class="my-login-form my-registration-form  col-sm-8">
+            <%--@elvariable id="driver" type="java"--%>
+                <form:form  action="/registration" method="post" modelAttribute="driver">
+                    <div class=" row">
+                        <div class="col-3"></div>
+                        <div class="col-9" style="color: #07fff6">
+                            <form:errors path="email"/>
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <label for="inputEmail" class="my-label-for-xs-resolution col-sm-3  col-form-label text-right" style="color: white; font-size: 15px;">Email</label>
+                        <div class="col-sm-9">
+                            <form:input path="email" type="email" class="form-control" id="inputEmail" placeholder="Input email"/>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-3"></div>
+                        <div class="col-9" style="color: #07fff6">
+                            <form:errors>${message}</form:errors>
+                            <form:errors path="password"/>
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <label for="inputPassword" class="my-label-for-xs-resolution col-sm-3  col-form-label text-right" style="color: white; font-size: 15px;">Password</label>
+                        <div class="col-sm-9 ">
+                            <form:password path="password" class="form-control" id="inputPassword"  placeholder="Input password"/>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-3"></div>
+                        <div class="col-9" style="color: #07fff6">
+                            <form:errors>${message}</form:errors>
+                            <form:errors path="repeatPassword"/>
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <label for="repeatPassword" class="my-label-for-xs-resolution col-sm-3  col-form-label text-right" style="color: white; font-size: 15px;">Repeat</label>
+                        <div class="col-sm-9 ">
+                            <form:password  path="repeatPassword" class="form-control" id="repeatPassword" placeholder="Repeate password"/>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-3"></div>
+                        <div class="col-9" style="color: #07fff6">
+                            <form:errors path="name"/>
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <label for="inputName" class="my-label-for-xs-resolution col-sm-3  col-form-label text-right" style="color: white; font-size: 15px;">Name</label>
+                        <div class="col-sm-9 ">
+                             <form:input path="name" class="form-control" id="inputName" placeholder="Input name"/>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-3"></div>
+                        <div class="col-9" style="color: #07fff6">
+                            <form:errors path="surname"/>
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <label for="inputSurname" class="my-label-for-xs-resolution col-sm-3  col-form-label text-right" style="color: white; font-size: 15px;">Surname</label>
+                        <div class="col-sm-9 ">
+                            <form:input path="surname" class="form-control" id="inputSurname" placeholder="Input surname"/>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-3"></div>
+                        <div class="col-9" style="color: #07fff6">
+                            <form:errors path="phone"/>
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <label for="inputPhone" class="my-label-for-xs-resolution col-sm-3  col-form-label text-right" style="color: white; font-size: 15px;">Phone</label>
+                        <div class="col-sm-9 ">
+                            <form:input type="phone" path="phone" class="form-control" id="inputPhone"  placeholder="Input phone"/>
+                        </div>
+                    </div>
+                    <div class="form-group row">
+                        <div class=" col-sm-12">
+                            <button type="submit" class="btn btn-success my-button-float-right">Sign Up</button>
+                        </div>
+                    </div>
+                </form:form>
+            </div>
+        </div>
+    </div>
+</body>
+</html>
\ No newline at end of file
Index: src/main/java/ua/repository/UserRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ua/repository/UserRepository.java	(date 1529930332696)
+++ src/main/java/ua/repository/UserRepository.java	(date 1529930332696)
@@ -0,0 +1,19 @@
+package ua.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import ua.entity.City;
+import ua.entity.User;
+
+import java.util.List;
+
+public interface UserRepository extends JpaRepository<User, Integer> {
+
+    User findByEmail(String email);
+
+    @Query("select c.name from City c")
+    List<String> findAllCitiesRepository();
+
+    @Query("select c from City c where c.id=?1")
+    City findCityByNameRepository(int id);
+}
Index: src/main/webapp/WEB-INF/views/edit.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/WEB-INF/views/edit.jsp	(date 1530429409718)
+++ src/main/webapp/WEB-INF/views/edit.jsp	(date 1530429409718)
@@ -0,0 +1,328 @@
+<%@ taglib prefix="a" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <style>
+        <%@include file='main-style.css' %>
+    </style>
+    <title>Edit</title>
+</head>
+<body style="background-image: url(http://ncwebcreations.com/wp-content/uploads/2016/12/pexels-photo-1-e1499782168666.jpg);">
+<nav class="navbar navbar-expand-lg navbar-light bg-light" >
+    <a class="navbar-brand " href="/">RentCar</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+            <sec:authorize access="isAuthenticated()">
+                <li class="nav-item">
+                    <a class="nav-link" href="/lend">Lend</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/borrow">Borrow</a>
+                </li>
+            </sec:authorize>
+            <sec:authorize access="hasRole('ROLE_ADMIN')">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="/admin" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Admin
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                        <a class="dropdown-item" href="/admin/brand">Brand</a>
+                        <a class="dropdown-item" href="/admin/city">City</a>
+                    </div>
+                </li>
+            </sec:authorize>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+            <sec:authorize access="isAnonymous()">
+                <li><a href="/registration"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+                <li><a href="/login"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+            </sec:authorize>
+            <sec:authorize access="isAuthenticated()">
+                <li><a href="/profile"><span class="glyphicon glyphicon-user"></span> Profile</a></li>
+                <form:form action="/logout">
+                    <a><button class="my-button-like-link"><span class="glyphicon glyphicon-log-in"></span> Logout</button></a>
+                </form:form>
+            </sec:authorize>
+        </ul>
+    </div>
+</nav>
+<div class="container">
+        <div class="my-class-for-picture">
+            <%--<img src="img/${driver.photo}">--%>
+            <%--@elvariable id="fileRequest" type="java"--%>
+            <form:form action="/profile/edit/picture" modelAttribute="fileRequest" enctype="multipart/form-data">
+                <input type="file" name="file">
+                <button>Ok</button>
+            </form:form>
+        </div>
+        <div class="row my-edit-about-driver">
+            <div class="col-sm-12"><h3>Driver</h3></div>
+            <div class="col-sm-6">
+                        <%--@elvariable id="driver" type="java"--%>
+                        <form:form  action="/profile/edit/info"  modelAttribute="driver">
+                            <div class="my-edit-info-form">
+                                <div class="form-group row">
+                                    <div class="col-9">
+                                        <p style="margin: 0px; color: white; font-size: 16px;">Change main info</p>
+                                    </div>
+                                    <div class=" col-3">
+                                        <button type="submit" class="btn btn-primary my-button-float-right">Change</button>
+                                    </div>
+                                </div>
+
+                                <div class="row">
+                                    <form:errors path="name" style="color: #07fff6"/>
+                                    <form:errors style="color: #07fff6">${message}</form:errors>
+                                </div>
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="name" for="name" cssStyle="color: white; float: right">Name</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+                                        <form:input path="name" class="form-control" id="name"  placeholder="Input new name"/>
+                                    </div>
+                                </div>
+
+                                <div class="row">
+                                    <form:errors path="surname" style="color: #07fff6"/>
+                                </div>
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="phone" for="surname" cssStyle="color: white; float: right">Surname</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+                                        <form:input  path="surname" class="form-control" id="surname" placeholder="Input new surname"/>
+                                    </div>
+                                </div>
+
+                                <div class="row">
+                                    <form:errors path="phone" style="color: #07fff6"/>
+                                </div>
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="phone" for="phone" cssStyle="color: white; float: right">Phone</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+                                         <form:input  path="phone" class="form-control" id="phone" placeholder="Input new number"/>
+                                    </div>
+                                </div>
+
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="placeOfBirth" for="place" cssStyle="color: white; float: right">City</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+                                         <form:select  path="placeOfBirth" items="${cities}" id="place" cssStyle="height: 33px" class="form-control"/>
+                                    </div>
+                                </div>
+
+                                <div class="row">
+                                    <form:errors path="dateOfBirth" style="color: #07fff6;"/>
+                                </div>
+
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="dateOfBirth" for="birth" cssStyle="color: white; float: right">Birthday</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+
+                                        <input  type="date" name="dateOfBirth" id="birth" class="form-control"/>
+                                    </div>
+
+                                </div>
+
+                                <div class="row">
+                                    <form:errors path="experienceBegan" style="color: #07fff6"/>
+                                </div>
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="experienceBegan" for="experienceBegan"  cssStyle="color: white; float: right">Start driving</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+                                        <input  type="date" name="experienceBegan" id="experienceBegan" class="form-control"/>
+                                    </div>
+                                </div>
+
+                            </div>
+                        </form:form>
+                    </div>
+
+                    <div class="col-sm-6">
+                        <%--@elvariable id="user" type="java"--%>
+                        <form:form  action="/profile/edit" method="post" modelAttribute="user">
+                            <div class="my-edit-info-form">
+                                <div class="form-group row">
+                                    <div class="col-9">
+                                        <p style="margin: 0px; color: white; font-size: 16px;">Change password</p>
+                                    </div>
+                                    <div class=" col-3">
+                                        <button type="submit"  class="btn btn-primary my-button-float-right">Change</button>
+                                    </div>
+                                </div>
+                                <div class="row">
+                                    <form:errors path="password" style="color: #07fff6"/>
+                                    <form:errors style="color: #07fff6">${message}</form:errors>
+                                </div>
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="password" for="inputPassword" cssStyle="color: white; float: right">New password</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+                                        <form:password path="password" class="form-control" id="inputPassword"  placeholder="Input new password"/>
+                                    </div>
+                                </div>
+
+                                <div class="row">
+                                    <form:errors path="repeatPassword" style="color: #07fff6"/>
+                                    <form:errors style="color: #07fff6">${message}</form:errors>
+                                </div>
+                                <div class="form-group row">
+                                    <div class="col-sm-4">
+                                        <form:label path="repeatPassword" for="repeatPassword" cssStyle="color: white; float: right">New password</form:label>
+                                    </div>
+                                    <div class="col-sm-8">
+                                        <form:password path="repeatPassword" class="form-control" id="repeatPassword"  placeholder="Input new password"/>
+                                    </div>
+                                </div>
+                            </div>
+                        </form:form>
+                </div>
+        </div>
+        <div class="row my-edit-about-car">
+            <div class="col-sm-12"><h2>Car</h2></div>
+            <div class="col-sm-6"></div>
+            <div class="col-sm-6">
+                <%--@elvariable id="car" type="java"--%>
+                <form:form  action="/profile/edit/car"  modelAttribute="car">
+                    <div class="my-edit-info-form">
+                        <div class="form-group row">
+                            <div class="col-9">
+                                <p style="margin: 0px; color: white; font-size: 16px;">Add or update car</p>
+                            </div>
+                            <div class=" col-3">
+                                <button type="submit"  class="btn btn-primary my-button-float-right">Update</button>
+                            </div>
+                        </div>
+
+
+                        <div class="row">
+                            <form:errors path="brand" style="color: #07fff6"/>
+                            <form:errors style="color: #07fff6">${message}</form:errors>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="brand" for="brand" cssStyle="color: white; float: right">Brand</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <form:select path="brand" items="${brands}" id="brand" class="form-control" cssStyle="height: 33px" />
+                            </div>
+                        </div>
+
+                        <div class="row">
+                            <form:errors path="model" style="color: #07fff6"/>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="model" for="model" cssStyle="color: white; float: right">Model</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <form:input path="model" class="form-control" id="model" placeholder="Input model"/>
+                            </div>
+                        </div>
+
+                        <div class="row">
+                            <form:errors path="yearOfManufacture" style="color: #07fff6"/>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="yearOfManufacture" for="yearOfManufacture"  cssStyle="color: white; float: right">Production</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <input  type="date" name="yearOfManufacture" id="yearOfManufacture" class="form-control"/>
+                            </div>
+                        </div>
+
+                        <div class="row">
+                            <form:errors path="body" style="color: #07fff6"/>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="body" for="body" cssStyle="color: white; float: right">Body</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <form:select  path="body" items="${bodies}" id="body" cssStyle="height: 33px" class="form-control"/>
+                            </div>
+                        </div>
+
+                        <div class="row">
+                            <form:errors path="engine" style="color: #07fff6"/>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="engine" for="engine" cssStyle="color: white; float: right">Engine</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <form:select  path="engine" items="${engines}" id="engine" cssStyle="height: 33px" class="form-control"/>
+                            </div>
+                        </div>
+
+                        <div class="row">
+                            <form:errors path="transmission" style="color: #07fff6"/>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="transmission" for="transmission" cssStyle="color: white; float: right">Transmission</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <form:select  path="transmission" items="${transmissions}" id="transmission" cssStyle="height: 33px" class="form-control"/>
+                            </div>
+                        </div>
+
+                        <div class="row">
+                            <form:errors path="drive" style="color: #07fff6"/>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="drive" for="drive" cssStyle="color: white; float: right">Drive</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <form:select  path="drive" items="${drives}" id="drive" cssStyle="height: 33px" class="form-control"/>
+                            </div>
+                        </div>
+
+                        <div class="row">
+                            <form:errors path="numberOfDoors" style="color: #07fff6"/>
+                        </div>
+                        <div class="form-group row">
+                            <div class="col-sm-4">
+                                <form:label path="numberOfDoors" for="numberOfDoors" cssStyle="color: white; float: right">Doors</form:label>
+                            </div>
+                            <div class="col-sm-8">
+                                <form:select  path="numberOfDoors" items="${doors}" id="numberOfDoors" cssStyle="height: 33px" class="form-control"/>
+                            </div>
+                        </div>
+                    </div>
+                </form:form>
+            </div>
+        </div>
+</div>
+
+</body>
+</html>
\ No newline at end of file
Index: src/main/webapp/WEB-INF/views/lend.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/WEB-INF/views/lend.jsp	(date 1530358615125)
+++ src/main/webapp/WEB-INF/views/lend.jsp	(date 1530358615125)
@@ -0,0 +1,175 @@
+<%@ taglib prefix="a" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <style>
+        <%@include file='main-style.css' %>
+    </style>
+    <title>Edit</title>
+</head>
+<body>
+<nav class="navbar navbar-expand-lg navbar-light bg-light" >
+    <a class="navbar-brand " href="/">RentCar</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav">
+            <sec:authorize access="isAuthenticated()">
+                <li class="nav-item">
+                    <a class="nav-link" href="/lend">Lend</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/borrow">Borrow</a>
+                </li>
+            </sec:authorize>
+            <sec:authorize access="hasRole('ROLE_ADMIN')">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" href="/admin" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        Admin
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                        <a class="dropdown-item" href="/admin/brand">Brand</a>
+                        <a class="dropdown-item" href="/admin/city">City</a>
+                    </div>
+                </li>
+            </sec:authorize>
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+            <sec:authorize access="isAnonymous()">
+                <li><a href="/registration"><span class="glyphicon glyphicon-user"></span> Sign Up</a></li>
+                <li><a href="/login"><span class="glyphicon glyphicon-log-in"></span> Login</a></li>
+            </sec:authorize>
+            <sec:authorize access="isAuthenticated()">
+                <li><a href="/profile"><span class="glyphicon glyphicon-user"></span> Profile</a></li>
+                <form:form action="/logout">
+                    <a><button class="my-button-like-link"><span class="glyphicon glyphicon-log-in"></span> Logout</button></a>
+                </form:form>
+            </sec:authorize>
+        </ul>
+    </div>
+</nav>
+
+<div class="container-for-lend">
+    <div class="my-lend-car">
+                <%--@elvariable id="car" type="java"--%>
+                <form:form  action="/lend" method="post"  modelAttribute="car">
+                    <div style="margin-bottom: 50px">
+                        <h2 style="color: white; text-align: center">Lend car to somebody who needs it </h2>
+                        <h4 style="color: white; text-align: center" >Your vehicle is able to earn money for you by itself</h4>
+                    </div>
+                    <div class="form-row">
+                        <form:errors style="color: #07fff6">${message}</form:errors>
+                    </div>
+                    <div class="form-row">
+                        <div class="form-group col-md-4 col-xs-6">
+                            <form:label path="region" for="region" cssStyle="color: white; float: right">City</form:label>
+                            <form:select  path="region" items="${cities}" id="region" cssStyle="height: 33px" class="form-control"/>
+                        </div>
+                        <div class="form-group col-md-2 col-xs-6">
+                            <form:label path="periodOfRentFrom" for="periodOfRentFrom"  cssStyle="color: white; float: right">Rent from</form:label>
+                            <input  type="date" name="periodOfRentFrom" id="periodOfRentFrom" class="form-control"/>
+                            <form:errors path="periodOfRentFrom" style="color: #07fff6"/>
+                        </div>
+
+                        <div class="form-group col-md-2 col-xs-6">
+                            <form:label path="periodOfRentTo" for="periodOfRentTo"  cssStyle="color: white; float: right">Rent to</form:label>
+                            <input  type="date" name="periodOfRentTo" id="periodOfRentTo" class="form-control"/>
+                            <form:errors path="periodOfRentTo" style="color: #07fff6"/>
+                        </div>
+
+                        <div class="form-group col-md-2 col-xs-6">
+                            <form:label path="pricePerDay" for="pricePerDay" cssStyle="color: white; float: right">Price</form:label>
+                            <form:input path="pricePerDay" class="form-control" id="pricePerDay"  placeholder="Input price"/>
+                            <form:errors path="pricePerDay" style="color: #07fff6"/>
+                        </div>
+
+                        <div class="form-group col-md-2 col-xs-6">
+                            <form:label path="chauffeur" for="chauffeur" cssStyle="color: white; float: right">Chauffeur</form:label>
+                            <form:select  path="chauffeur" items="${chauffeurs}" id="chauffeur" cssStyle="height: 33px" class="form-control"/>
+                        </div>
+                        <div class="form-group col-md-12 col-xs-6">
+                            <button type="submit"  class="btn btn-success my-button-float-right">Add car</button>
+                        </div>
+                    </div>
+                </form:form>
+    </div>
+</div>
+
+
+<div class="my-form-for-particular-order col-md-7 col-xs-12">
+    <h2 style="color: white">Particular order</h2>
+    <div class="my-particular-order">
+        City: <b>${particularOrder.region}</b> |
+        Rent from: <b>${particularOrder.periodOfRentFrom}</b> |
+        Rent to: <b>${particularOrder.periodOfRentTo}</b> |
+        Price: <b>${particularOrder.pricePerDay}</b> |
+        Chauffeur: <b>${particularOrder.chauffeur}</b> |
+        <a href="/lend/delete/${particularOrder.id}" class="btn btn-warning btn-sm">Delete</a>
+    </div>
+    <div style="height:310px; overflow:auto;">
+        <a:forEach items="${clients}" var="client">
+            <div class="my-list-of-clients col-md-6 col-xs-6">
+                <div class="my-margin-for-list-of-clients">
+                    <img style="width: 100px; float: right; display: block; color: white;" src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100">
+                    <h4 style="color: white;">${client.name} ${client.surname}</h4>
+                    <h5 style="margin: 2px 10px; color: white;">${client.phone}</h5>
+                    <h5 style="margin: 2px 10px; color: white;">${client.age} years old, ${client.placeOfBirth}</h5>
+                    <h5 style="margin: 2px 10px; color: white;">${client.experience} year(s) of practice</h5>
+                    <h5 style="margin: 2px 10px; color: white;">${client.countOfTrips} finished trip(s)</h5>
+                    <a href="/lend/refuse/${client.id}/${particularOrder.id}" style="float: right; margin: 2px" class="btn btn-warning btn-sm">Refuse</a>
+                    <a href="/lend/confirm/${client.id}/${particularOrder.id}" style="float: right; margin: 2px" class="btn btn-primary btn-sm">Confirm</a>
+                </div>
+            </div>
+        </a:forEach>
+
+        <a:forEach items="${reservedOrder}" var="client">
+            <div class="my-list-of-clients col-md-6 col-xs-6">
+                <div class="my-margin-for-client">
+                    <img style="width: 100px; float: right; display: block; color: white;" src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100">
+                    <h4 style="color: white;">${client.name} ${client.surname}</h4>
+                    <h5 style="margin: 2px 10px; color: white;">${client.phone}</h5>
+                    <h5 style="margin: 2px 10px; color: white;">${client.age} years old, ${client.placeOfBirth}</h5>
+                    <h5 style="margin: 2px 10px; color: white;">${client.experience} year(s) of practice</h5>
+                    <h5 style="margin: 2px 10px; color: white;">${client.countOfTrips} finished trip(s)</h5>
+                    <a href="/lend/complete/${particularOrder.id}" style="float: right; margin: 2px" class="btn btn-warning btn-sm">Complete</a>
+                </div>
+            </div>
+        </a:forEach>
+    </div>
+</div>
+
+<div class="my-form-for-finished-orders col-md-5 col-xs-12">
+        <div class="form-row">
+            <h2 style="color: white">Finished trips</h2>
+        </div>
+    <div style="height:390px; overflow:auto;">
+        <a:forEach items="${finishedOrders}" var="order">
+            <div class="my-particular-order">
+                <a style="float: right" href="/driver/${order.driverId}" class="btn btn-warning btn-sm">Driver</a>
+                City: <b>${order.region}</b> |
+                Price: <b>${order.pricePerDay}</b> |
+                Chauffeur: <b>${order.chauffeur}</b> <br>
+                Rent from: <b>${order.periodOfRentFrom}</b> |
+                Rent to: <b>${order.periodOfRentTo}</b>
+            </div>
+        </a:forEach>
+    </div>
+</div>
+
+</body>
+</html>
\ No newline at end of file
